addons/modules/functions/fnc_tracers.sqf

```diff
@@ -0,0 +1,163 @@
+#include "script_component.hpp"
```

Comment:
Rename this function to something like `moduleTracersServer`.

Reply:
12e8b52